api.js

```diff
@@ -59,10 +72,9 @@ apiRouter.get('/gltf', (req, res) => {
     try {
         const resp = await (partId ? TranslationService.translatePart(req.user.accessToken, gltfElemId, partId, translationParams)
             : TranslationService.translateElement(req.user.accessToken, gltfElemId, translationParams));
-        const data = (await resp.json()).data;
-        const contentType = resp.headers.get('Content-Type');
-        res.status(200).contentType(contentType).send(data);
+        res.status(200).contentType(resp.contentType).send(resp.data);
     } catch (err) {
+        console.log(`GET /gltf: error: ${err}`);
```

Comment:
This looks like debug logging? Remove it please.

Reply:
Good catch! Removed.